### ISSUE TITLE
gopkg.in/jarcoal/httpmock.v1 currently breaks application using NeverbounceApi-Go

### ISSUE THREAD

**Issue by richardsabow** [NeverBounce/NeverBounceApi-Go#26]
My application is using the go wrapper of neverbounce API, but after upgrading from go 1.12 to go 1.15 the build fails (even when I call `go mod tidy`) because of httpmock.v1 dependency. The error is:

```
github.com/NeverBounce/NeverBounceApi-Go tested by
github.com/NeverBounce/NeverBounceApi-Go.test imports
gopkg.in/jarcoal/httpmock.v1: no matching versions for query "latest"
```

Even when I call manually `go get gopkg.in/jarcoal/httpmock.v1` the result will be the same. 

The issue could be resolved by replacing the imports of `httpmock.v1` from `gopkg.in/jarcoal/httpmock.v1` to `github.com/jarcoal/httpmock`. Also would be worth to add `go.mod` to the package.
Do you think it's feasible? Or do you have any other suggestion for resolving this issue?

**Comment by pjarmalavicius** [comment]
Any news about solving this issue?

**Comment by louiscollarsmith** [comment]
Seeing the same issue too.

**Comment by alexcarol** [comment]
> Any news about solving this issue?

I sent an email to neverbounce support, and they mentioned that they are actively working on the request, but do not have an ETA. As they don't seem to be tracking github issues, I'd recommend submitting a ticket to support too.

**Comment by mmollick** [comment]
Hey all, thanks for the patience with this. I just merged in #30, which introduces the `go.mod`, updates the `httpmock` dependency, and enabled Github Actions (previously using Travis) with versions 1.15 and 1.16.

I'm going to close this issue, but feel free to re-open it if necessary.
